sway-lib-core/src/primitive_conversions.sw

```diff
@@ -141,6 +235,50 @@ impl u64 {
 }
 
 impl u32 {
+    /// Extends a `u32` to a `u64`.
+    ///
+    /// # Returns
+    ///
+    /// * [u64] - The converted `u32` value.
+    ///
+    /// # Examples
+    ///
+    /// ```sway
+    /// use std::assert::assert;
+    ///
+    /// fn foo() {
+    ///     let val = 10u32;
+    ///     let result = val.as_u64();
+    ///     assert(result == 10);
+    /// }
+    /// ```
+    pub fn as_u64(self) -> u64 {
```

Comment:
is this a new function added? Should be in a seperate pr with testing

Reply:
Oh no... I think this is an artifact from the master branch